package.json

```
@@ -167,7 +167,7 @@
     "got": "^11.8.3",
     "gray-matter": "^4.0.3",
     "jiti": "^2.1.0",
-    "knip": "^5.21.0",
+    "knip": "5.30.6",
```

```suggestion
    "knip": "^5.32.0",
```
knip version [`5.32.0`](https://github.com/webpro-nl/knip/releases/tag/5.32.0) has been released, and I don't see any issues that are referenced on the ESLint issue thread: - https://github.com/eslint/eslint/issues/18990 Can we proceed with migrating directly to knip `^5.32.0`?